### DISCUSSION TITLE
Issue about recompiling codes in HPC

### DISCUSSION THREAD

**Discussion by hokonkim** [idaholab/moose#18177] [Q&A General] [answered]
Hello!

After I developed several code for MOOSE, I uploaded the code in HPC.

While I tried to recompile the code, I got errors like below.

![image](https://user-images.githubusercontent.com/46009457/123482690-933d0e80-d5d3-11eb-9eb9-748a04edbdb1.png)

So far, even I got an error about "gfortran", but actually compiling itself was done successfully.

But, this time, I don't know why this happened.

Is there any suggestions for me?

Thanks!

**Comment by GiudGiud** [comment, marked as answer]
Hello

Looks like you need to make sure you have MPI wrapped compilers available. Depending on which cluster you are working on, they would be available in modules. You would need to load those. 

If there are no modules and no one set up moose for you on the cluster, then you can follow these instructions:
https://mooseframework.inl.gov/getting_started/installation/hpc_install_moose.html

Guillaume

**Reply by hokonkim** [reply]
Because I didn't work in HPC so long, I forgot to load module which I needed in HPC.

After loading modules in HPC, recompiling itself was done successfully.

Thanks!
